### PR TITLE
Translation of 2020-methodology to Japanese

### PR DESCRIPTION
**Pull request by ksakae1216**
I translate, Please review.
part of #1460

### CODE REVIEW

**Comment by ksakae1216** [comment]
@bazzadp @MSakamaki 
Sorry, I will fix error, wait please.

**Comment by bazzadp** [comment]
> @bazzadp @MSakamaki
> Sorry, I will fix error, wait please.

No worries - that's what the linter is for! Still looks broken btw even after you last commit.

File: src/templates/ja/2020/methodology.html [hunk 1]
```diff
@@ -360,39 +365,85 @@ <h3 id="rework-css"><a href="#rework-css" class="anchor-link">Rework CSS</a></h3
...
+          </li>
+          <li>
+            2019年の執筆者の多くはGoogleの社員であり、今年はより広いウェブコミュニティからの視点をよりバランスよく得ることを試みました。Web Alamanacの声はコミュニティそのものを反映したものであるべきであり、反響室を作らないように特定の企業に偏ったものではないと考えています。
+          </li>
+          <li>
+            プロジェクトのリーダーは、すべての著者の推薦を検討し、新しい視点をもたらし、地域社会の中での未発表グループの声を増幅させる著者の選定に尽力しました。
+          </li>
+        </ul>
+
+        将来的には、このプロセスを繰り返していくことで、Web Almanacがより多様で包括的なプロジェクトとなり、さまざまなバックグラウンドを持つ投稿者が参加できるようにしたいと考えています。
```
Comment (bazzadp) [inline src/templates/ja/2020/methodology.html:426]: Ah forgot to say that we had to fix this code in #1500 

```suggestion
        そのために、今年は著者の探し方、選び方を体系的に変えてみました。
      </p>
      
      <ul>
        <li>
          これまでの著者は、異なる視点のためのスペースを確保するために、再び書くことを特別に望まれていませんでした。
        </li>
        <li>
          2020年の著者を支持する人たちは皆、見た目や考え方が似ている人を指名しないように特に意識するように求められました。
        </li>
        <li>
          2019年の執筆者の多くはGoogleの社員であり、今年はより広いウェブコミュニティからの視点をよりバランスよく得ることを試みました。Web Alamanacの声はコミュニティそのものを反映したものであるべきであり、反響室を作らないように特定の企業に偏ったものではないと考えています。
        </li>
        <li>
            プロジェクトのリーダーは、すべての著者の推薦を検討し、新しい視点をもたらし、地域社会の中での未発表グループの声を増幅させる著者の選定に尽力しました。
        </li>
      </ul>
      
      <p>
        将来的には、このプロセスを繰り返していくことで、Web Almanacがより多様で包括的なプロジェクトとなり、さまざまなバックグラウンドを持つ投稿者が参加できるようにしたいと考えています。
```

**Review by bazzadp — CHANGES_REQUESTED** [review]
One small subsequent change, other than that looks good.

You can commit my suggestion in GitHub if happy with it.

File: src/templates/ja/2020/methodology.html [hunk 2]
```diff
@@ -240,7 +245,7 @@ <h3 id="webpagetest"><a href="#webpagetest" class="anchor-link">WebPageTest</a><
       </p>
 
       <p>
-        HTTP ArchiveのWebPageTestのプライベートインスタンスは、最新のパブリックバージョンと同期して維持され、<a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">カスタムメトリクス</a>で強化されています。これらは、テストの最後に各ウェブサイトで評価されるJavaScriptのスニペットです。<a href="https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/almanac.js">almanac.js</a>カスタムメトリクスには、DOMの状態に依存するメトリクスなど、そうでなければ計算できないいくつかのメトリクスが含まれています。
+        HTTP ArchiveのWebPageTestのプライベートインスタンスは、最新のパブリックバージョンと同期して維持され、<a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">カスタムメトリクス</a>で強化されています。これらは、テストの最後に各ウェブサイトで評価されるJavaScriptのスニペットです。多くのデータアナリストの<a href="https://github.com/HTTPArchive/legacy.httparchive.org/commits/master/custom_metrics">貢献</a>のおかげで、特に <a href="https://github.com/HTTPArchive/legacy.httparchive.org/pulls?q=is%3Apr+author%3ATiggerito+sort%3Acreated-asc"><a href="{{ url_for('contributors', year=year, lang=lang, _anchor='Tiggerito') }}">Tony McCreath</a>の献身的な努力</a>により、Web Almanacの2020年版では、HTTPアーカイブのテストインフラストラクチャの機能が3,000行以上の新しいコードで大幅に拡張されました。
```
Comment (bazzadp) [inline src/templates/ja/2020/methodology.html:248]: This is still not right. You can't have a link inside of a link.

File: src/templates/ja/2020/methodology.html [hunk 3]
```diff
@@ -240,7 +245,7 @@ <h3 id="webpagetest"><a href="#webpagetest" class="anchor-link">WebPageTest</a><
       </p>
 
       <p>
-        HTTP ArchiveのWebPageTestのプライベートインスタンスは、最新のパブリックバージョンと同期して維持され、<a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">カスタムメトリクス</a>で強化されています。これらは、テストの最後に各ウェブサイトで評価されるJavaScriptのスニペットです。<a href="https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/almanac.js">almanac.js</a>カスタムメトリクスには、DOMの状態に依存するメトリクスなど、そうでなければ計算できないいくつかのメトリクスが含まれています。
+        HTTP ArchiveのWebPageTestのプライベートインスタンスは、最新のパブリックバージョンと同期して維持され、<a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">カスタムメトリクス</a>で強化されています。これらは、テストの最後に各ウェブサイトで評価されるJavaScriptのスニペットです。多くのデータアナリストの<a href="https://github.com/HTTPArchive/legacy.httparchive.org/commits/master/custom_metrics">貢献</a>に感謝します、特に<a href="https://github.com/HTTPArchive/legacy.httparchive.org/pulls? q=is%3Apr+author%3ATiggerito+sort%3Acreated-asc">献身的な努力</a>をした<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='Tiggerito')}}">Tony McCreathのおかげで</a>、Web Almanacの2020年版では、HTTPアーカイブのテストインフラストラクチャの機能が3,000行以上の新しいコードで大幅に拡張されました。
```
Comment (bazzadp) [inline src/templates/ja/2020/methodology.html:248]: There's a space in there that shouldn't be there:

```suggestion
        HTTP ArchiveのWebPageTestのプライベートインスタンスは、最新のパブリックバージョンと同期して維持され、<a href="https://github.com/HTTPArchive/legacy.httparchive.org/tree/master/custom_metrics">カスタムメトリクス</a>で強化されています。これらは、テストの最後に各ウェブサイトで評価されるJavaScriptのスニペットです。多くのデータアナリストの<a href="https://github.com/HTTPArchive/legacy.httparchive.org/commits/master/custom_metrics">貢献</a>に感謝します、特に<a href="https://github.com/HTTPArchive/legacy.httparchive.org/pulls?q=is%3Apr+author%3ATiggerito+sort%3Acreated-asc">献身的な努力</a>をした<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='Tiggerito')}}">Tony McCreathのおかげで</a>、Web Almanacの2020年版では、HTTPアーカイブのテストインフラストラクチャの機能が3,000行以上の新しいコードで大幅に拡張されました。
```

Comment (ksakae1216) [reply src/templates/ja/2020/methodology.html]: Thanks, I fix.

File: src/templates/ja/2020/methodology.html (hunk 1 above)
Comment (ksakae1216) [reply src/templates/ja/2020/methodology.html:426]: OK, I fix.

**Review by bazzadp — APPROVED** [review]
Just one tiny indentation fix and then all good from myside. I'll go ahead and commit that.

@MSakamaki let us know if you'll get a chance to review otherwise will merge this before the next release.

File: src/templates/ja/2020/methodology.html [hunk 4]
```diff
@@ -360,39 +365,87 @@ <h3 id="rework-css"><a href="#rework-css" class="anchor-link">Rework CSS</a></h3
...
+          <li>
+            2020年の著者を支持する人たちは皆、見た目や考え方が似ている人を指名しないように特に意識するように求められました。
+          </li>
+          <li>
+            2019年の執筆者の多くはGoogleの社員であり、今年はより広いウェブコミュニティからの視点をよりバランスよく得ることを試みました。Web Alamanacの声はコミュニティそのものを反映したものであるべきであり、反響室を作らないように特定の企業に偏ったものではないと考えています。
+          </li>
+          <li>
+            プロジェクトのリーダーは、すべての著者の推薦を検討し、新しい視点をもたらし、地域社会の中での未発表グループの声を増幅させる著者の選定に尽力しました。
+          </li>
+        </ul>
```
Comment (bazzadp) [inline src/templates/ja/2020/methodology.html:425]: Nit - indentation is too much:

```suggestion
      <ul>
        <li>
          これまでの著者は、異なる視点のためのスペースを確保するために、再び書くことを特別に望まれていませんでした。
        </li>
        <li>
          2020年の著者を支持する人たちは皆、見た目や考え方が似ている人を指名しないように特に意識するように求められました。
        </li>
        <li>
          2019年の執筆者の多くはGoogleの社員であり、今年はより広いウェブコミュニティからの視点をよりバランスよく得ることを試みました。Web Alamanacの声はコミュニティそのものを反映したものであるべきであり、反響室を作らないように特定の企業に偏ったものではないと考えています。
        </li>
        <li>
          プロジェクトのリーダーは、すべての著者の推薦を検討し、新しい視点をもたらし、地域社会の中での未発表グループの声を増幅させる著者の選定に尽力しました。
        </li>
      </ul>
```

**Comment by MSakamaki** [comment]
@ksakae1216 @bazzadp 
I about to start my review!

File: src/templates/ja/2020/methodology.html [hunk 5]
```diff
@@ -360,39 +365,87 @@ <h3 id="rework-css"><a href="#rework-css" class="anchor-link">Rework CSS</a></h3
...
+            これは、セレクタと特異性に関連するすべてのCSSメトリクスで広く使われています。
+        </p>
+    </section>
+
     <section>
       <h2 id="process"><a href="#process" class="anchor-link">分析プロセス</a></h2>
 
       <p>
-        Web Almanacの計画と実行には、Webコミュニティの数十人の貢献者の協力を得て、約1年を要しました。このセクションでは、Web Almanacに掲載されているメトリクスを選択した理由、それらがどのようにして照会されたか、そして解釈されたかについて説明します。
+        Web Almanacの計画と実行には、100人以上のWebコミュニティの<a href="{{ url_for('contributors', year=year, lang=lang) }}">投稿者</a>の協力を得て、約1年を要しました。このセクションでは、Web Almanacに掲載されているチャプターを選んだ理由、それらのメトリクスがどのようにして照会されたか、そしてどのように解釈されたかを説明します。
```
Comment (MSakamaki) [inline src/templates/ja/2020/methodology.html:389]: @ksakae1216 Other translate, a term is unified.
```suggestion
        Web Almanacの計画と実行には、100人以上のWebコミュニティの<a href="{{ url_for('contributors', year=year, lang=lang) }}">貢献者</a>の協力を得て、約1年を要しました。このセクションでは、Web Almanacに掲載されている章を選んだ理由、それらのメトリクスがどのようにして分析されたか、そしてどのように解釈されたかを説明します。
```

File: src/templates/ja/2020/methodology.html [hunk 6]
```diff
@@ -360,39 +365,87 @@ <h3 id="rework-css"><a href="#rework-css" class="anchor-link">Rework CSS</a></h3
...
       </p>
     </section>
 
     <section>
-      <h3 id="brainstorming"><a href="#brainstorming" class="anchor-link">ブレインストーミング</a></h3>
+      <h3 id="planning"><a href="#planning" class="anchor-link">プランニング</a></h3>
+
+      <p>
+        2020年のWeb Almanacは、<a href="https://twitter.com/rick_viscomi/status/1273135952848977920">2020年6月</a>にキックオフしましたが、COVID-19に関連した不安や社会正義の抗議活動のため、2019年のタイムラインよりも遅くなりました。2020年のこれらの出来事やその他の出来事は、制作プロセス全体の下火となり、このようなテンポの速いプロジェクトのストレスを超えて、私たちの貢献者に多くの追加的な負担をかけました。
```
Comment (MSakamaki) [inline src/templates/ja/2020/methodology.html:397]: @ksakae1216 
kicked off = 開始、発端
https://eow.alc.co.jp/search?q=kick+off
```suggestion
        2020年のWeb Almanacは、<a href="https://twitter.com/rick_viscomi/status/1273135952848977920">2020年6月</a>に開始されましたが、COVID-19に関連した不安や社会正義の抗議活動のため、2019年のタイムラインよりも遅くなりました。2020年のこれらの出来事やその他の出来事は、制作プロセス全体の下火となり、このようなテンポの速いプロジェクトのストレスを超えて、私たちの貢献者に多くの追加的な負担をかけました。
```

**Review by MSakamaki — COMMENTED** [review]
@ksakae1216 I review completed, please confirm!

File: src/templates/ja/2020/methodology.html [hunk 7]
```diff
@@ -20,17 +20,22 @@
         </li>
         <li><a href="#tools">ツール</a>
           <ul>
-            <li><a href="#webpagetest">WebPageTest</a></li>
+            <li><a href="#webpagetest">ウェブページテスト</a></li>
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/templates/ja/2020/methodology.html:23]: I fix.

File: src/templates/ja/2020/methodology.html [hunk 8]
```diff
@@ -20,17 +20,22 @@
...
+                <li><a href="#rework-utils">Rework Utils</a></li>
+              </ul>
+            </li>
+            <li><a href="#parsel">Parsel</a></li>
           </ul>
         </li>
         <li><a href="#process">分析プロセス</a>
           <ul>
-            <li><a href="#brainstorming">ブレインストーミング</a></li>
+            <li><a href="#planning">プランニング</a></li>
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/templates/ja/2020/methodology.html:38]: I fix.

File: src/templates/ja/2020/methodology.html [hunk 9]
```diff
@@ -54,15 +59,15 @@
...
         </p>
 
         <p>
-          Web Almanacの使命は、主題の専門家が文脈に沿った洞察を提供することで、HTTP Archiveのデータウェアハウスをウェブコミュニティにさらにアクセスしやすくすることです。2020年が第2版となる2020年は、ウェブの状態に関する知識の年次リポジトリと考えることができます。
+          Web Almanacの使命は、ウェブの状態に関する一般的な知識の年次リポジトリになることです。私たちの目標は、主題の専門家が文脈に沿った洞察を提供することで、HTTP Archiveのデータウェアハウスをウェブコミュニティがさらに利用しやすいものにすることです。
         </p>
 
         <p>
-          Web Almanacの2020年版は、コンテンツ、体験、パブリッシング、ディストリビューションの4つの柱で構成されています。書かれたレポートの各パートは柱を表し、その異なる側面を探求する章で構成されています。例えば、パートIIはユーザー体験を表し、パフォーマンス、セキュリティ、アクセシビリティ、SEO、PWA、モバイルウェブの各章が含まれています。
+          Web Almanacの2020年版は、コンテンツ、体験、パブリッシング、ディストリビューションの4つのパートに分かれています。各パートの、いくつかの章では、その全体的なテーマをさまざまな角度から探求しています。例えば、パートIIでは、パフォーマンス、セキュリティ、アクセシビリティの各章で、ユーザー体験をさまざまな角度から探求しています。
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/templates/ja/2020/methodology.html:70]: I fix.

File: src/templates/ja/2020/methodology.html [hunk 10]
```diff
@@ -360,39 +365,87 @@ <h3 id="rework-css"><a href="#rework-css" class="anchor-link">Rework CSS</a></h3
...
+        今年の<a href="{{ url_for('chapter', year=year, lang=lang, chapter='css') }}">CSS</a>の章では、<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='LeaVerou') }}">Lea Verou</a>がCSSの状態をかなり詳細に見ており、<a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020/01_CSS">100以上のクエリ</a>にわたっています。視点を変えれば、2019年の2.5倍のクエリ数になります。この規模の分析を実現可能にするために、Leaは<a href="https://github.com/LeaVerou/rework-utils">Rework Utils</a>をオープンソース化しました。これらのユーティリティは、CSSの洞察をより簡単に抽出するための有用なスクリプトを提供することで、ReworkからJSONデータを次のレベルに引き上げます。CSSの章で見る統計情報のほとんどは、これらのスクリプトによって提供されています。
+      </p>
+    </section>
+
+    <section>
+        <h4 id="parsel"><a href="#parsel" class="anchor-link">Parsel</a></h4>
+
+        <p>
+            <a href="https://projects.verou.me/parsel/">Parsel</a>はCSSセレクタパーサとspecificity calculatorで、元々は<a href="{{ url_for('chapter', year=year, lang=lang, chapter='css') }}">CSS</a>の章をリードする<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='LeaVerou') }}">Lea Verou</a>と別のライブラリとしてオープンソース化されています。
+            これは、セレクタと特異性に関連するすべてのCSSメトリクスで広く使われています。
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/templates/ja/2020/methodology.html:381]: I fix.

File: src/templates/ja/2020/methodology.html [hunk 11]
```diff
@@ -360,39 +365,87 @@ <h3 id="rework-css"><a href="#rework-css" class="anchor-link">Rework CSS</a></h3
...
+      <p>
+        今年の<a href="{{ url_for('chapter', year=year, lang=lang, chapter='css') }}">CSS</a>の章では、<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='LeaVerou') }}">Lea Verou</a>がCSSの状態をかなり詳細に見ており、<a href="https://github.com/HTTPArchive/almanac.httparchive.org/tree/main/sql/2020/01_CSS">100以上のクエリ</a>にわたっています。視点を変えれば、2019年の2.5倍のクエリ数になります。この規模の分析を実現可能にするために、Leaは<a href="https://github.com/LeaVerou/rework-utils">Rework Utils</a>をオープンソース化しました。これらのユーティリティは、CSSの洞察をより簡単に抽出するための有用なスクリプトを提供することで、ReworkからJSONデータを次のレベルに引き上げます。CSSの章で見る統計情報のほとんどは、これらのスクリプトによって提供されています。
+      </p>
+    </section>
+
+    <section>
+        <h4 id="parsel"><a href="#parsel" class="anchor-link">Parsel</a></h4>
+
+        <p>
+            <a href="https://projects.verou.me/parsel/">Parsel</a>はCSSセレクタパーサとspecificity calculatorで、元々は<a href="{{ url_for('chapter', year=year, lang=lang, chapter='css') }}">CSS</a>の章をリードする<a href="{{ url_for('contributors', year=year, lang=lang, _anchor='LeaVerou') }}">Lea Verou</a>と別のライブラリとしてオープンソース化されています。
```
[the remark this answers is not in the document]
Comment (ksakae1216) [reply src/templates/ja/2020/methodology.html:380]: I fix.

File: src/templates/ja/2020/methodology.html (hunk 5 above)
Comment (ksakae1216) [reply src/templates/ja/2020/methodology.html:389]: I fix.

File: src/templates/ja/2020/methodology.html (hunk 6 above)
Comment (ksakae1216) [reply src/templates/ja/2020/methodology.html:397]: I fix.

**Review by MSakamaki — APPROVED** [review]
💯

**Comment by bazzadp** [comment]
Great work - thanks both!

**Pull request merged**